packages/backend/server/src/__tests__/__snapshots__/mailer.spec.ts.md

```diff
@@ -1403,9 +1490,9 @@ Generated by [AVA](https://avajs.dev).
                       >Test Workspace</span␊
                     >␊
                     team workspace expired on<!-- -->␊
-                    <span style="font-weight:600">1/1/2025</span>. All workspace␊
+                    <span style="font-weight:600">2025/1/1</span>. All workspace␊
                     data will be permanently deleted on<!-- -->␊
-                    <span style="font-weight:600">1/31/2025</span> (180 days after␊
+                    <span style="font-weight:600">2025/1/31</span> (180 days after␊
```

Comment:
required <blockquote>👀 <a href="https://app.graphite.dev//github/pr/toeverything/AFFiNE/10916?ref=gt-pasteable-stack">#10916 feat(server): display date in yyyy-MM-dd format on email</a> <code>+38/-26</code></blockquote>